src/proteus/atmos_clim/agni.py

```diff
@@ -360,8 +364,8 @@ def run_agni(atmos, loops_total:int, dirs:dict, OPTIONS:dict, hf_row:dict):
 
         # Try solving temperature profile
         agni_success = jl.AGNI.solver.solve_energy_b(atmos,
-                            sol_type=OPTIONS["atmosphere_surf_state"],
-                            chem_type=OPTIONS["atmosphere_chemistry"],
```

Comment:
The parameter `chem_type` also needs to be passed as an integer.
It should either be `0` when the config variable is `None`, or `1` otherwise.